dashboard/src/components/SearchBox.js

```diff
@@ -0,0 +1,79 @@
+import React from "react";
+import PropTypes from "prop-types";
+
+import { withStyles } from "material-ui/styles";
+import Paper from "material-ui/Paper";
+// import Typography from "material-ui/Typography";
+import filterIcon from "material-ui-icons/FilterList";
+
+const styles = theme => ({
+  box: {
+    display: "flex",
+    border: "1px solid",
+    borderColor: theme.palette.divider,
+  },
+  filterIconContainer: {
+    display: "flex",
+    padding: "6px 8px 8px",
+    height: 36,
+    color: theme.palette.action.active,
+  },
+  textField: {
+    borderRadius: 3,
+    width: 300,
+    height: 36,
+    fontSize: 14,
```

Comment:
🤔 we'll have to go through and make sure _all_ our text consistent at some point.